app/src/main/java/com/github/se/icebreakrr/ui/sections/shared/DisplayProfile.kt

```diff
@@ -187,7 +190,14 @@ fun ProfileHeader(
                             .background(IceBreakrrBlue, CircleShape),
                     contentAlignment = Alignment.Center) {
                       IconButton(
-                          onClick = { onEditClick() }, modifier = Modifier.testTag("editButton")) {
+                          onClick = {
+                            if (isNetworkAvailable(context = context)) {
+                              onEditClick()
+                            } else {
+                              showNoInternetToast(context = context)
```

Comment:
One improvement that we can do later is link that with the caching. That means that if I edit my profile with no connection, I want to cache it and update firestore once my wifi comes back